data_collection/gazette/spiders/mg_belo_horizonte.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+import re
+
+import scrapy
+from dateutil.rrule import DAILY, rrule
+from urllib.parse import urlencode
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgBeloHorizonteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3106200"
+
+    name = "mg_belo_horizonte"
+    allowed_domains = ["dom-web.pbh.gov.br"]
+    start_date = datetime.datetime(2000, 1, 2)
+    end_date = None
+
+    def start_requests(self):
+        base_url = "https://api-dom.pbh.gov.br/api/v1/edicoes/buscarpublicacaopordata"
+
+        end_date = self.end_date or datetime.date.today()
```

Comment:
I'm guessing this was intended to not make a conflict with `scrapy crawl mg_belo_horizonte -a end_date=YYYY-MM-DD` right? If that's the only reason, I just checked and the base class is already overriding the class attribute if the argument is present
 
```suggestion
    end_date = datetime.date.today()

    def start_requests(self):
        base_url = "https://api-dom.pbh.gov.br/api/v1/edicoes/buscarpublicacaopordata"
```